### DISCUSSION TITLE
Constellation complemented by Coconut SVSM

### DISCUSSION THREAD

**Discussion by gonzalez-cy** [edgelesssys/constellation#3389] [General]
Hi there,

I've recently discovered the Coconut SVSM and I was wondering if it makes sense to install it so it can complement Constellation. If doing this would create some kind of duplicity, I would like to know.

**Comment by thomasten** [comment]
Hi,
Yes, this would make perfect sense. But currently, the CVM's firmware is under the control of the cloud provider. This is true for all providers that Constellation supports.
If this changes in the future, we'll consider using Coconut SVSM with Constellation. We're actually involved in its development.

**Reply by gonzalez-cy** [reply]
Nice, I'll keep an eye on this. Thank you for the feedback =)
